src/components/Datatable/ToolsTabItem/ToolsTabItem.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { isNotUndefined, noop } from 'ramda-adjunct';
+import styled from 'styled-components';
+
+import { SpacingSizeValuePropType } from '../../../types/spacing.types';
+import { getColor, pxToRem } from '../../../utils/helpers';
+import { IconTypes, SSCIconNames } from '../../Icon/Icon.enums';
+import { SSCIcons, Types } from '../../Icon/Icon.types';
+import Button from '../../Button/Button';
+import StyledIcon from '../../Button/StyledIcon';
+import { ButtonSizes } from '../../Button/Button.enums';
+import { ToolsTabItemProps } from './ToolsTabItem.types';
+import { ToolsTabItemStates } from './ToolsTabItem.enum';
+
+const DefaultToolsTabItem = styled(Button)`
+  &&& {
```

Comment:
This shouldn't be needed

Reply:
I would actually recreate this from HTML `button` tag. You probably wont need all the functionality from `Button` component (like dynamic tag resolution). Since this will be used just for the tooltab where all tools will be defined by the `Datatable` component itself so it doesn't need to be that flexible. And if we decide later to add this variant in the `Button` component we would still need styled directly there.

Reply:
This makes more sense, using html `button` instead now